tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -9042,6 +9056,15 @@ SpirvEmitter::processIntrinsicCallExpr(const CallExpr *callExpr) {
   case hlsl::IntrinsicOp::IOP_firstbitlow: {
     retVal = processIntrinsicFirstbit(callExpr, GLSLstd450::GLSLstd450FindILsb);
     break;
+  }
+  case hlsl::IntrinsicOp::IOP_isnan: {
```

Comment:
Should this be done for `isinf` as well?

Reply:
Yes, `isinf` is a problem as well. Please open a new issues for `isinf`, no need to hold up this PR.

https://godbolt.org/z/fKfzaxE6E

Reply:
I opened the follow up issue. https://github.com/microsoft/DirectXShaderCompiler/issues/6783

Reply:
Oh right, thanks. Yes we might want to refactorize a bit this, several intrinsics might be impacted.